crates/ruff_linter/src/rules/pycodestyle/rules/logical_lines/missing_whitespace_around_operator.rs

```diff
@@ -150,13 +151,15 @@ pub(crate) fn missing_whitespace_around_operator(
         }
 
         match kind {
+            TokenKind::FStringStart => fstrings += 1,
+            TokenKind::FStringEnd => fstrings = fstrings.saturating_sub(1),
             TokenKind::Lpar | TokenKind::Lambda => parens += 1,
             TokenKind::Rpar => parens = parens.saturating_sub(1),
             _ => {}
         };
 
-        let needs_space = if kind == TokenKind::Equal && parens > 0 {
-            // Allow keyword args or defaults: foo(bar=None).
+        let needs_space = if kind == TokenKind::Equal && (parens > 0 || fstrings > 0) {
+            // Allow keyword args, defaults: foo(bar=None) and f-strings: f'{foo=}'
```

Comment:
Again, should we just have something like:

```rust
if fstrings > 0 {
    continue;
}
```

Or is it important that we run this body logic?

Reply:
Oh, so we're trying to catch _some_ of these errors within f-strings, but not all -- is that right?

Reply:
Yes, it's only when the equal sign is present which in f-string context will be used as debug expression. So, `f"{x=}` and `f"{x = }"` gives different output.